BlueShift-iOS-SDK/InApps/BlueShiftNotificationWebViewController.m

```diff
@@ -185,7 +185,14 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
         NSURL *url = navigationAction.request.URL;
         [self sendActionEventAnalytics: url.absoluteString];
         
-        if([BlueShift sharedInstance].appDelegate.oldDelegate && [[BlueShift sharedInstance].appDelegate.oldDelegate respondsToSelector:@selector(application:openURL:options:)]) {
+        if (self.inAppNotificationDelegate && [self.inAppNotificationDelegate respondsToSelector:@selector(actionButtonDidTapped:)]) {
+            NSMutableDictionary *actionPayload = [[NSMutableDictionary alloc] init];
```

Comment:
Looks like there is an indentation problem with all the following lines. Please check and correct if there is one.